pulp_rpm/app/urls.py

```diff
@@ -1,8 +1,8 @@
-from django.conf.urls import url
-
-from .viewsets import CopyViewSet
-
-
+# from django.conf.urls import url
```

Comment:
i suggest to remove this file completely.
even if we bring back the copy endpoint it most likely will live somewhere under /repositories/rpm/rpm/<id>/copy/